flintrock/scripts/setup-ephemeral-storage.py

```diff
@@ -159,6 +160,23 @@ def create_root_dir():
     return path
 
 
+def create_tmp_dir(devices):
```

Comment:
Selecting `devices[0]` a few lines down smells a bit to me. I think this API would be clearer if it simply accepted the path that `/media/tmp` should be pointed at. `__main__` should be tasked with selecting the appropriate path to pass to this function.